modules/microshift-get-rpm-release-info.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+//microshift_updating/microshift-list-update-contents.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-rpm-release-info_{context}"]
+= List the contents of the {microshift-short} RPM release package
+
+To see the images included with a {microshift-short} release, you can list the contents of the `microshift-release-info` RPM by downloading and unpacking the RPM.
+
+.Prerequisites
+
+* You are using {op-system-full} or an operating system with an RPM package manager.
```

Comment:
Something looks incorrect here. The word is not formatting correctly from what I am seeing